pkg/operator/resource/resourceapply/generic.go

```diff
@@ -53,27 +54,28 @@ func ApplyDirectly(client kubernetes.Interface, recorder events.Recorder, manife
 		}
 		result.Type = fmt.Sprintf("%T", requiredObj)
 
+		// NOTE: Do not add CR resources into this switch otherwise the protobuf client can cause problems.
```

Comment:
or explicitly use `kubeClient` instead

Reply:
i don't think we ever want to add CR's here, I think this message is good enough to scare people from doing it